File: app/src/main/java/com/example/loginapp/Adapter/ViewPagerAdapter.java
package com.example.loginapp.Adapter;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import com.example.loginapp.Fragments.Contact_Us;
import com.example.loginapp.Fragments.Images;
import com.example.loginapp.Fragments.View_Images;

public class ViewPagerAdapter extends FragmentPagerAdapter {

    public ViewPagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        switch (position) {
            case 0:
                return new Contact_Us(); //ChildFragment1 at position 0
            case 1:
                return new Images(); //ChildFragment2 at position 1
            case 2:
                return new View_Images(); //ChildFragment3 at position 2
        }
        return null; //does not happen
    }

    @Override
    public int getCount() {
        return 3; //three fragments
    }

    @Override
    public CharSequence getPageTitle(int position) {
        String title = getItem(position).getClass().getName();
        title = title.replaceAll("_", " ");

        return title.subSequence(title.lastIndexOf(".") +1, title.length());
    }
}

File: app/src/main/java/com/example/loginapp/Adapter/ViewPagerAdapter.kt
package com.example.loginapp.Adapter

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter
import com.example.loginapp.Fragments.Contact_Us
import com.example.loginapp.Fragments.Images
import com.example.loginapp.Fragments.View_Images


class ViewPagerAdapter(fm: FragmentManager?) : FragmentPagerAdapter(fm!!) {
    override fun getItem(position: Int): Fragment {
        when (position) {
            0 -> return Contact_Us() //ChildFragment1 at position 0
            1 -> return Images() //ChildFragment2 at position 1
            2 -> return View_Images() //ChildFragment3 at position 2
        }
        return Contact_Us() //does not happen
    }

    override fun getCount(): Int {
        return 3 //three fragments
    }

    override fun getPageTitle(position: Int): CharSequence? {
        var title = getItem(position).javaClass.name
        title = title.replace("_".toRegex(), " ")
        return title.subSequence(title.lastIndexOf(".") + 1, title.length)
    }
}